src/main/java/net/openhft/chronicle/bytes/AbstractBytes.java

```diff
@@ -1321,6 +1322,17 @@ public int byteCheckSum(@NonNegative int start, @NonNegative int end)
         return sum & 0xFF;
     }
 
+    protected boolean isBackedByEmptyByteStore() {
+        return EmptyByteStore.class.isInstance(bytesStore);
```

Comment:
I think it would be more performant (and certainly more object-oriented) if ByteStore had a default isEmptyBytesStore() returning false and EmptyByteStore had it overridden and return true - then you may remove this method

Reply:
The problem is that AbstractBytes is in the public API available for anyone to override. But we could certainly override it for some implementations.

Reply:
It is hard to override this method in any Bytes implementation as many are elastic and one can provide any initial ByteStore.

Reply:
If it's a default method with sensible default, no action is required from implementors.